sdk/search/azure-search-documents/azure/search/documents/_internal/_index_documents_batch.py

```diff
@@ -47,11 +51,12 @@ def add_upload_actions(self, *documents):
         :param documents: Documents to upload to an Azure search index. May be
          a single list of documents, or documents as individual parameters.
         :type documents: dict or list[dict]
+        :rtype: List[IndexAction]
```

Comment:
None of your tests are using this return type (if I read correctly). Does it mean we need more tests, or that finally return them here is not necessary? Thoughts?

Reply:
Added :)